compiler/src/Language/Bond/Codegen/Cpp/Grpc_h.hs

```diff
@@ -270,50 +246,44 @@ inline #{className}::#{proxyName}<TThreadPool>::#{proxyName}(
 
         methodDecl Function{..} = [lt|#{template}template <typename TThreadPool>
 inline void #{className}::#{proxyName}<TThreadPool>::Async#{methodName}(
-    ::std::shared_ptr< ::grpc::ClientContext> context,
     #{voidParam (methodTypeToMaybe methodInput)}
-    const std::function<void(std::shared_ptr< ::bond::ext::gRPC::unary_call_result< #{payload (methodTypeToMaybe methodResult)}>>)>& cb)
+    const std::function<void(std::shared_ptr< ::bond::ext::gRPC::unary_call_result< #{payload (methodTypeToMaybe methodResult)}>>)>& cb,
```

Comment:
missing ::std at two places

Reply:
Intentionally did not update to avoid more changes.